codes/quantum/properties/symmetry_protected.yml

```diff
@@ -0,0 +1,31 @@
+code_id: symmetry_protected
+
+name: 'Symmetry-protected self-correcting quantum code'
+introduced: '\cite{arXiv:1805.01474}'
+
+description: |
+  The physical degrees of freedom of the code correspond to a physical lattice with finite-dimensional local Hilbert space. The codespace is the degenerate ground space of a Hamiltonian that consists of terms localized in space. The system is required to have a symmetry not spontaneously broken, with the symmetry-protected self-correcting property described as follows.
```

Comment:
Remember that audience is not always physicists. Need to define what a "symmetry" is here and what does it mean to respect i (i.e., some operator U commuting with Ham). What kinds of U are allowed? You can then use this to help with the text below, like defining what it means for the bath to respect symmetry.